### ISSUE TITLE
new HMDHFDplus version

### ISSUE THREAD

**Issue by timriffe** [willekens/VirtualPop#5]
Hi Frans! HMDHFDplus will soon have a new version on CRAN, which
1) fixes the UTF-8 tail problem you pointed out in the HFD data
2) now refers to the new HMD website and requires new HMD credentials.

The new version number will be 2.0.0; your code shouldn't need to change, unless you had a manual fix for the UTF-8 thing. You'll only need to specify a new minimum HMDHFDplus version number in your `DESCRIPTION` file. I expect the new version to be on CRAN in 2-3 days. All the best,
Tim

**Comment by willekens** [comment]
Dear Tim

Thank you.

With best wishes

Frans
________________________________
From: Tim Riffe ***@***.***>
Sent: 01 February 2023 22:54
To: willekens/VirtualPop ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [willekens/VirtualPop] new HMDHFDplus version (Issue #5)


Hi Frans! HMDHFDplus will soon have a new version on CRAN, which

  1.  fixes the UTF-8 tail problem you pointed out in the HFD data
  2.  now refers to the new HMD website and requires new HMD credentials.

The new version number will be 2.0.0; your code shouldn't need to change, unless you had a manual fix for the UTF-8 thing. You'll only need to specify a new minimum HMDHFDplus version number in your DESCRIPTION file. I expect the new version to be on CRAN in 2-3 days. All the best,
Tim

**Comment by timriffe** [comment]
Of course, I've now made a pull request that addresses this and other things. CRAN noticed that your package would break if HMDHFDplus were updated as requested, so I took care of that here and there, you will have received a notification. Note the min package dependency is 2.0.01, and so the remote checks using github actions (not sure if you are using these or not, but I have them on my fork) won't be able to install VirtualPop to test it until HMDHFDplus is finally on CRAN. Sorry if that's confusing, but it should be sorted out in a day or two
